docs/getting-started/getting-started-with-docker.rst

```diff
@@ -22,15 +24,143 @@ There is a good chance that you have heard of containerization. A container is e
+
+Tutorial: Downloading and running an existing Docker image
+----------------------------------------------------------
+
+Finding an official image
+~~~~~~~~~~~~~~~~~~~~~~~~~
+Let's start with Docker Hub. From the homepage, we can click "Explore" in the top righthand corner in order to get to `the search page <https://hub.docker.com/search?q=>`__, allowing us to start digging through all of its images. Don't worry, it won't load all 9,200,012 images at once.
+
+First, let's click the "Docker Official Image" option on the left, then search "Python" using the search bar at the top. Fittingly, our top result is the official Python image.
+
+.. image:: /assets/images/docs/docker/dockerhub-search-python.png
+    :alt: Screenshot of the Docker Hub search feature, with the official Python image showing up as the first result
+
+Upon clicking on that result, we are taken to a description page. There is a lot of information here, such as an explanation of the multiple variants of the image. But what we're interested in right now is the command in the black box towards the top right. Assuming you have `installed Docker <https://docs.docker.com/desktop/linux/install/>`__ already -- now is a good time to do that -- this is the command used to pull the image from Docker Hub's servers into your computer, allowing you to execute it locally.
+
+.. image:: /assets/images/docs/docker/dockerhub-python-descript.png
+    :alt: Screenshot of the Docker Hub's page for Python, which has the gray text "Copy and paste to pull this image" above a black box containing the words "docker pull python"
+
+Let's try that next!
+
+Downloading an image from Docker Hub
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Once Docker is installed, open up the command line and enter the following:
+
+::
+
+    $> docker pull python
+
+The response will, at first, look a bit like this:
+
+.. code:: bash
+    
+    Using default tag: latest
+    latest: Pulling from library/python
+    e756f3fdd6a3: Pull complete
+    bf168a674899: Pull complete
+    e604223835cc: Pull complete
+    6d5c91c4cd86: Extracting [>                                                  ]  557.1kB/54.58MB
+    2cc8d8854262: Downloading [========================================>          ]  159.8MB/196.7MB
+    2767dbfeeb87: Download complete
+    9d5e973c5e10: Download complete
+    45f2aca7694f: Download complete
+    1123e010bf80: Download complete
+
+This shows the progress of downloading each :ref:`dict layer` included in the image, one at a time, plus extracting them. Once a layer is downloaded and extracted, it will read "Pull complete" in the CLI. This whole process usually takes just a few seconds. In this case, it creates the Python Docker image.
```

Comment:
> This whole process usually takes just a few seconds

I just tried this at the office, with our super-fast connection, and fully pulling the Python image took 30-60 seconds for me.

Reply:
I thought I had cleared my cache before this download but just in case I adjusted it. Thanks for testing this out and catching this!